Makefile

```diff
@@ -94,3 +111,33 @@ generate-client: ## Generate client from latest swagger file
 	echo "git add client" && \
 	echo "git commit -m 'Bump swagger client to version $$SWAGGER_VERSION'"
 	@rm -rf ./gen-swagger
+
+.PHONY: ecr-connect
+ecr-connect: ## Login to ecr, requires aws cli installed
+	aws configure set region $(AWS_REGION)
+	aws configure set aws_access_key_id $(AWS_ACCESS_KEY_ID)
+	aws configure set aws_secret_access_key $(AWS_SECRET_ACCESS_KEY)
```

Comment:
I think you can remove all those configure parts, they are not required if env vars are exported